src/config/config.go

```diff
@@ -101,17 +103,17 @@ func GetArch(archs ...string) string {
 	return runtime.GOARCH
 }
 
-// Timestamp of when the CLI was started
+// GetStartTime Timestamp of when the CLI was started.
```

Comment:
should be `GetStartTime returns the timestamp of when...`

Reply:
Fixed